projects/widgets/docs/getting-started.md

```diff
@@ -1,3 +1,66 @@
-# getting started
+# Getting started
 
-...
+## 1. Registering a widget project
+
+Before you can start building your widget you must make a widget project on our [self-service platform Projectaanvraag](https://projectaanvraag.uitdatabank.be). The registration process is very simple and only takes a few minutes.
+
+The created widget project will use event data from our test environment. No payment is required until you decide you want to use real event data.
+
+<https://www.youtube.com/watch?v=X0uplBw98gg>
+
+<!-- theme: success -->
+
+> Once you have created your widget project, you're all set to start building your widget!
+
+## 2. Building your widget
+
+### Choosing a template 🗂️
+
+To help you get started quickly, we offer 4 different templates:
+
+1. **Leeg**: You start from a blanc layout that you then expand yourself by adding rows and assigning blocks (e.g. search box or search results) to these rows.
```

Comment:
```suggestion
1. **Leeg**: You start from a blank layout that you then expand yourself by adding rows and assigning blocks (e.g. search box or search results) to these rows.
```

At least I think it should be `blank`. See https://en.wiktionary.org/wiki/blanc#English

If you have a definition that explains that it should be `blanc` in some cases (in English), do share!